.github/workflows/ci-unix-shared-installed.yml

```diff
@@ -0,0 +1,65 @@
+# This is a copy of ci-unix-shared-local.yml for building shared libraries, but
+# using installed dependencies instead of locally built ones.
```

Comment:
The original purpose of ci-unix-shared-local.yml is to test whether we can build libavif itself as a shared library and export all the public API functions from the shared library correctly. Whether libavif also uses the dependencies as shared libraries doesn't really matter to the check of public API function exports. And whether the dependencies are installed on the system or built locally doesn't really matter either. So this new CI build is redundant for the original purpose, but it is useful for testing an additional build configuration, and for that purpose it would be good to test using dav1d as the AV1 decoder.